src/protocols/secure_channel/CASEServer.h

```diff
@@ -61,6 +61,8 @@ class CASEServer : public SessionEstablishmentDelegate,
     Messaging::ExchangeManager * mExchangeManager        = nullptr;
     SessionResumptionStorage * mSessionResumptionStorage = nullptr;
 
+    Optional<SessionHandle> mCaseSession;
+
     CASESession mPairingSession;
```

Comment:
I have no good suggestions, but this naming leaves something to be desired.

Reply:
CASESession -> mCaseProtocolHandler